resources/index.html

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>Available users</title>
+    </head>
+    <body>
+        <script type="text/javascript" src="./client scripts/index.js"></script>
```

Comment:
It's better to link script files in the head section, and have a small inlined code that will create a class/call a function defined in your JS file.